views/auth/store.ejs

```diff
@@ -0,0 +1,183 @@
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta
+            name="viewport"
+            content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <!-- define links -->
+    <link rel="stylesheet" href="/css/store.css">
+    <!-- import Bootstrap 5-->
+    <link rel="stylesheet" href="/sources/bootstrap.css">
+    <title>Store</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css">
+
+</head>
+<body>
+<div class="container">
```

Comment:
sen container'in sedece 48 satıra kadar kullandın, tüm proje uygularsan sağ sol boşluk problemi çözülecek.
yani 168 satıra koy, ve yaptığın marginler ve padding sil gerek kalmayacak